src/modules/chat_scrollback_settings/index.js

```diff
@@ -0,0 +1,47 @@
+const watcher = require('../../watcher');
+const storage = require('../../storage');
+const twitch = require('../../utils/twitch');
+
+const DEFAULT_SIZE = 200;
+const PROMPT_BODY = `Change the number of messages the chat is able to scrollback to. Increasing this value can slow down your browser...
+Leaving this value empty restore default settings.
```

Comment:
this sentence is not proper grammar

Reply:
I've changed it in my new commit. If it's still not good, help me please, english isn't my first language :)